.github/workflows/cluster-test/mysql/dolphinscheduler_env.sh

```diff
@@ -0,0 +1,48 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# JAVA_HOME, will use it to start DolphinScheduler server
+export JAVA_HOME=${JAVA_HOME:-/usr/local/openjdk-8}
```

Comment:
could we use set .env in docker instead of using the whole file? I think is a little duplicate from the original `dolphinscheduler_env.sh` file

Reply:
> could we use set .env in docker instead of using the whole file? I think is a little duplicate from the original `dolphinscheduler_env.sh` file

```start-all.sh``` will ```souce dolphinscheduler_env.sh``` which will overwrite all ```.env``` in docker. So I think it's necessary.